pkgdb/src/resolver/manifest-raw.cc

```diff
@@ -646,6 +646,25 @@ HookRaw::check() const
     }
 }
 
+/* -------------------------------------------------------------------------- */
+
+void
+from_json( const nlohmann::json & jfrom, BuildDescriptorRaw & build )
+{
+  // for building we only need the command,
+  // other attribute are handled by `flox` and passed to the build script as
+  // applicable
+  auto value = jfrom.at( "command" );
+  value.get_to( build.command );
+}
+
+void
+to_json( nlohmann::json & jto, const BuildDescriptorRaw & build )
+{
+  jto            = nlohmann::json::object();
+  jto["command"] = build.command;
```

Comment:
question nonblocking: when are we serializing the manifest? It's a bit surprising to have a lossy roundtrip. Can we just drop `to_json`?

Reply:
we still have serialization code for `ManifestRaw`.

Reply:
Ok since i dont think we are parsing v1 manifests as manifest raw anywhere but `BuildenvLockfile::from_v1_content`, let alone write it with pkgdb, i removed the serializing/deserializing from manifest raw and removed the `to_json` impl for `BuildDescriptorRaw` altogether

Reply:
Nice! Did you forget to commit that? 🤔 https://github.com/flox/flox/pull/1802/files#diff-ce87e2d882355bd1745e04b94e8e01e781bb6f85b1e3542b482c2b8a48702dfdR347

Reply:
yeah forgot about the header 🫠

Reply:
removed the header part